module ProvideException

exception OriginalException

let excVal1 = OriginalException

provide { exception OriginalException as MyException, excVal1 }
